storage/src/main/java/org/apache/kafka/storage/internals/log/LogConfig.java

```diff
@@ -102,9 +102,9 @@ public String topicWarningMessage(String topicName) {
 
     public static class RemoteLogConfig {
 
-        private final boolean remoteStorageEnable;
-        private final long localRetentionMs;
-        private final long localRetentionBytes;
+        public final boolean remoteStorageEnable;
+        public final long localRetentionMs;
+        public final long localRetentionBytes;
```

Comment:
I believe we already have public accessor functions in LogConfig for these.

See LogConfig.localRetentionMs(), LogConfig.localRetentionBytes() and LogConfig.remoteStorageEnable()